src/components/ExportButton/ExportButton.js

```diff
@@ -80,15 +80,39 @@ function ExportButton({
           setLoading(false);
           return;
         }
-        exportPdf(
+
+        const { exportConfig } = topic;
+        let imageUrl;
+        let fileName = `trafimage-${new Date().toISOString().slice(0, 10)}.pdf`;
+
+        if (exportConfig) {
+          const { overlayImageUrl, exportFileName } = exportConfig;
+          imageUrl =
+            overlayImageUrl &&
+            (typeof overlayImageUrl === 'function'
+              ? overlayImageUrl(i18n.language)
+              : overlayImageUrl);
+
+          fileName = exportFileName
```

Comment:
make it always a function getOverlayImgUrl, it's simplifies

Reply:
done